.gitignore

```diff
@@ -28,3 +28,6 @@ src/**/components.d.ts
 # backup files
 *.bak
 *.orig
+
+# Docs Preview
+__docs-temp__/*
```

Comment:
Good catch! Can you merge master and move this under `# Compiled files`?

Reply:
done